R/implicit_integer_linter.R

```diff
@@ -9,6 +9,11 @@
 #'   linters = implicit_integer_linter()
 #' )
 #'
+#' lint(
+#'   text = "x[c(1, 2)]",
```

Comment:
I would stick with a simpler example, `x[1]` -> `x[1L]`, the `c()` part here makes it look like an example for `c()`, not for extraction